src/NumberFormat/NumberFormat.php

```diff
@@ -67,13 +67,27 @@ final class NumberFormat
      */
     protected $decimalSeparator = '.';
 
+    /**
+     * The decimal separator for currency.
```

Comment:
We're not consistent. The variable doc say s"The decimal separator for currency." but the getter says "Gets the currency decimal separator." Let's standardize either on "currency decimal separator" or "decimal separator for currency amounts". Leaning towards the second one, but your choice.

Same applies for the grouping separator.

Reply:
Good catch, thanks!